### PR TITLE
Add job list for localization id_ID

### LINKED ISSUE #337: jobTitle() returns random value for localization id_ID
### Summary

jobTitle() is not returning expected value, because the list of jobs are not available for localization id_ID. I already make pull request for fixing this issue.

### Versions

|                | Version |
|:---------------|:--------|
| PHP            | 8.0.8   |
| `fakerphp/faker`  | 1.14.0   |

### Self-enclosed code snippet for reproduction

```php
require 'vendor/autoload.php';
$faker = Faker\Factory::create('id_ID');
echo $faker->jobTitle();
```

### Expected output

```txt
# return job titles in Indonesian
```

### Actual output

```txt
# return random value of string
```

### PR DESCRIPTION
**Pull request by CikiMomogi**
Add job list for id_ID

### What is the reason for this PR?

- [ ] A new feature
- [x] Fixed an issue (resolve #337)

### Author's checklist

- [x] Follow the [Contribution Guide](https://github.com/FakerPHP/Faker/blob/main/.github/CONTRIBUTING.md)
- [ ] New features and changes are [documented](https://github.com/FakerPHP/fakerphp.github.io)

### Summary of changes

Add list of jobs for indonesian. All test were passed succesfully ([here](https://github.com/CikiMomogi/Faker/pull/1))

### Review checklist

- [ ] All checks have passed
- [ ] Changes are approved by maintainer

### CODE REVIEW

**Comment by CikiMomogi** [comment]
All checks have [passed](https://github.com/CikiMomogi/Faker/pull/1)

File: src/Faker/Provider/id_ID/Company.php [hunk 1]
```diff
@@ -16,6 +16,16 @@ class Company extends \Faker\Provider\Company
      */
     protected static $companyPrefix = ['PT', 'CV', 'UD', 'PD', 'Perum'];
 
+    /**
+     * @see https://www.amesbostonhotel.com/macam-macam-profesi-pekerjaan/
+     */
+    protected static $jobTitleFormat = [
```
Comment (pimjansen) [inline src/Faker/Provider/id_ID/Company.php:22]: Can we sort the names as mention in the contribution guide?

**Review by pimjansen — CHANGES_REQUESTED** [review]

File: src/Faker/Provider/id_ID/Company.php (hunk 1 above)
Comment (CikiMomogi) [reply src/Faker/Provider/id_ID/Company.php:23]: Done sorted the list

**Review by bramceulemans — APPROVED** [review]
Looks good to me now!

**Review by pimjansen — APPROVED** [review]

**Comment by pimjansen** [comment]
Thanks for contributing!

**Pull request merged**